python/src/scipp/plotting/resampling_model.py

```diff
@@ -101,8 +100,12 @@ def _call_resample(self):
                 else:
                     # cannot pre-select bins for multi-dim coords
                     if len(out.meta[dim].dims) == 1:
-                        out = out[sc.get_slice_params(out.data, out.meta[dim],
-                                                      low, high)]
+                        # TODO: for now, slicing in scipp does not support step
+                        # and pybind11 py::slice requires a step
+                        # (which is always 1 here)
+                        sl = sc.get_slice_params(out.data, out.meta[dim], low,
+                                                 high)
```

Comment:
Would it be a better fix to just change `get_slice_params` to return with a step of 1 (to be honest I still don't understand why this change is needed now, was something broken?)?

Reply:
It because before we were going through the `_maybe_slice` mechanism where we were extracting the slice's `start` and `stop`: https://github.com/scipp/scipp/pull/1907/files#diff-f7dd39172a47ac585ef2a5c0714a8f5ba6acb848db46f59cdea519f4706a8091L73
(I think we were always in bin edges?)

The `py::slice` object always has a step, to it tries to slice the DataArray with `5:10:1` and we don't support the step of 1 in scipp, hence the TODO.

Reply:
I don't understand. We *only* support `step=1`.

Reply:
Yes but you can't write `d['tof', 5:10:1]`, you have to write `d['tof', 5:10]`.

Reply:
The first one raises an exception

Reply:
Hmm, works for variables. Can you just fix this, so it also works for data arrays?

Reply:
ok, i didn't know it worked for Variables!

Reply:
The issue appears to be that this wrongly selects some value-based slicing overload, rather than positional.